### DISCUSSION TITLE
how to do it

### DISCUSSION THREAD

**Discussion by mrjo3000** [3kh0/ext-remover#701] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how to do it?

**Comment by bartholemuew** [comment, marked as answer]
how to do WHAT?!?!? you people are incapable of elaborating on ANYTHING, you should stop looking for unblocking methods and start paying attention to your 4th-grade teacher.

**Reply by AshtonDavies** [reply]
Fax

**Reply by 3kh0** [reply]
> how to do WHAT?!?!? you people are incapable of elaborating on ANYTHING, you should stop looking for unblocking methods and start paying attention to your 4th-grade teacher.

this ^^
